src/pages/ProfilesExplorerView/components/SceneAiPanel/infrastructure/ProfileApiClient.ts

```diff
@@ -0,0 +1,38 @@
+import { TimeRange } from '@grafana/data';
+
+import { DataSourceProxyClient } from '../../../infrastructure/series/http/DataSourceProxyClient';
+
+// dot format returns string (TODO: json format later)
```

Comment:
It all depends.

For instance, this one has been tackled in [this PR](https://github.com/grafana/explore-profiles/pull/132/files#diff-f75b69b1b716ea9aaa14b38dadc1a2fc28fb48532caa3381adf764bacdb0736cL11).

But generally speaking, because we gave more importance to speed and flexibility, we were not strict. So far, I've used private reminders to do some code maintenance whenever I could. I now believe we should have some kind of process. Let's talk!